package.json

```diff
@@ -71,7 +76,6 @@
     "@types/react-dom": "^18.0.0",
     "@typescript-eslint/eslint-plugin": "^2.18.0",
     "@typescript-eslint/parser": "^2.18.0",
-    "@wessberg/rollup-plugin-ts": "^1.2.15",
```

Comment:
The author of this plugin has published new versions of this plugin as `rollup-plugin-ts`